wordpress-mobile/WordPress-iOS
# Comments: readability and autosizing

This is a leftover area for implementing readability and autosizing. Fairly straightforward, nuked the custom layout from before and added the usual implementation changes, including stackViews for the cell.

To test:
1. Open up Comments for a site. Blog Details > Comments.
2. Check that the margins and cell heights are as expected.
3. Check with iPad, iPhone and iPhone Plus.
4. Check with landscape and portrait orientations.
5. Check with iPad multitasking states.

Needs review: @frosty can I peg you for one more readability PR? 😃 (no rush)
